### ISSUE TITLE
Nginx proxy buffering creates useless I/O on server

### ISSUE THREAD

**Issue by ldperron** [SUSE/Portus#1753]
In this line:
https://github.com/SUSE/Portus/blob/9ee98db159fa469951d009bcaecee19a6643bade/examples/compose/nginx/nginx.conf#L140

Since docker images are large objects, nginx proxy buffering needs temporary files to handle buffering, which leads to useless disk I/O. In our setup, it actually reduces the performances.

**Comment by Vad1mo** [comment]
true, the best setup would be like the one from https://github.com/jwilder/nginx-proxy I has pretty good defaults.

**Comment by mssola** [comment]
True. Thanks for the heads up!
